src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  background-color: #f4f4f4;
+  font-family: Roboto, Arial, sans-serif;
+
+  --accent-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo-link {
+  display: flex;
+}
+
+.nav-links {
+  list-style: none;
```

Comment:
You have used a tag name (`ul`) for styling. This is not recommended. Instead, you should use class names. You could add a class to the `ul` element in your HTML and use that class for styling. For example: `.nav-list { list-style: none; }`